### ISSUE TITLE
custom_claims must be configurable through GUI

### ISSUE THREAD

**Issue by dibericky** [primait/localauth0#27]
In [this PR](https://github.com/primait/localauth0/pull/24) support to custom claims in access token has been added. 
This feature should be available also through the GUI.

**Comment by cottinisimone** [comment]
Yes, moreover i would find a way to add `permissions` field to custom fields too being that that field is a custom field in auth0 for our (prima.it) use case
